src/close_fds.c

```diff
@@ -74,10 +73,8 @@ void close_other_fds()
 {
 	int fd;
 
-	for (fd = 3; fd < open_files_max_fd; fd++) {
+	for (fd = 3; fd <= open_files_max_fd; fd++) {
 		if (open_files_set == NULL || FD_ISSET(fd % FD_SETSIZE, &(open_files_set[fd / FD_SETSIZE])))
```

Comment:
sorry, one more quick nit, can we return from this function early if `open_files_set == NULL`?

Reply:
good idea!

Pushed new version